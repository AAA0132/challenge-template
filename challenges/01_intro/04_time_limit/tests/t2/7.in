217739223100244648550
