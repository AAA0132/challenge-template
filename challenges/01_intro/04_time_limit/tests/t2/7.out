23705184638149056390318441043006630875525
